composer.json

```diff
@@ -2,12 +2,10 @@
     "name": "elgentos/largeconfigproducts",
     "description": "",
     "require": {
-        "php": "~5.5.0|~5.6.0|~7.0.0|~7.1.0|~7.2.0",
-        "renatocason/magento2-module-mq": "^1.0",
-        "renatocason/magento2-module-mq-mysql": "^1.1",
-        "magento/module-configurable-product": "^100.2.8"
+        "php": "~7.1.3||~7.2.0||~7.3.0",
```

Comment:
Could you add 7.4 as well? Afaik there are no forward incompatible changes.

Reply:
Done!